Dockerfiles/manifests/agent-only/secrets.yaml

```diff
@@ -0,0 +1,12 @@
+# This file has been generated by `helm template datadog-agent stable/datadog` from datadog/templates/secrets.yaml. Please re-run `generate.sh` rather than modifying this file manually.
+# API Key
+apiVersion: v1
+kind: Secret
+metadata:
+  name: datadog-agent
+  labels: {}
+type: Opaque
+data:
+  api-key: "PERBVEFET0dfQVBJX0tFWT4="
```

Comment:
is it a random value?

Reply:
This one isn’t random. It’s the base64-encoded version of the default value in the `values.yaml` of the public chart:
```
echo -n PERBVEFET0dfQVBJX0tFWT4= | base64 -d
<DATADOG_API_KEY>
```

The end-user will anyhow need to customise this value.
I can add an extra`sed` to make it more explicit.